Plugins/SequencePlugin/Source/SequencePlugin/Public/Marketplace/Marketplace_Enums.h

```diff
@@ -0,0 +1,131 @@
+};
+
+UENUM(BlueprintType)
+enum EOrderStatus
+{
+    UNKNOWN_OST UMETA(DisplayName = "Unknown"),
+    ACTIVE  UMETA(DisplayName = "Active"),
+    INACTIVE  UMETA(DisplayName = "Inactive"),
+    EXPIRED  UMETA(DisplayName = "Expired"),
+    CANCELLED  UMETA(DisplayName = "Cancelled"),
+    FILLED  UMETA(DisplayName = "Filled"),
+};
+//
+//
+//// PropertyType Extensions
+//UCLASS()
+//class UPropertyTypeExtensions : public UObject
+//{
+//    GENERATED_BODY()
+//
+//public:
+//    UFUNCTION(BlueprintCallable, Category = "PropertyType")
+//    static FString AsString(EPropertyType PropertyType)
+//    {
+//        switch (PropertyType)
+//        {
+//        case EPropertyType::INT:     return "INT";
+//        case EPropertyType::STRING:  return "STRING";
+//        case EPropertyType::ARRAY:   return "ARRAY";
+//        case EPropertyType::GENERIC: return "GENERIC";
+//        default: return "Unknown";
+//        }
+//    }
+//};
+//
+//
+//// MarketplaceKind Extensions
+//UCLASS()
+//class UMarketplaceKindExtensions : public UObject
+//{
+//    GENERATED_BODY()
+//
+//public:
+//    UFUNCTION(BlueprintCallable, Category = "MarketplaceKind")
+//    static FString AsString(EMarketplaceKind MarketplaceKind)
+//    {
+//        switch (MarketplaceKind)
+//        {
+//        case EMarketplaceKind::UNKNOWN:                return "unknown";
+//        case EMarketplaceKind::SEQUENCE_MARKETPLACE_V1: return "sequence_marketplace_v1";
+//        case EMarketplaceKind::SEQUENCE_MARKETPLACE_V2: return "sequence_marketplace_v2";
+//        case EMarketplaceKind::OPENSEA:                return "opensea";
+//        case EMarketplaceKind::MAGIC_EDEN:             return "magic_eden";
+//        case EMarketplaceKind::MINTIFY:                return "mintify";
+//        case EMarketplaceKind::LOOKS_RARE:             return "looks_rare";
+//        case EMarketplaceKind::X2Y2:                   return "x2y2";
+//        case EMarketplaceKind::SUDO_SWAP:              return "sudo_swap";
+//        case EMarketplaceKind::COINBASE:               return "coinbase";
+//        case EMarketplaceKind::RARIBLE:                return "rarible";
+//        case EMarketplaceKind::NFTX:                   return "nftx";
+//        case EMarketplaceKind::FOUNDATION:             return "foundation";
+//        case EMarketplaceKind::MANIFOLD:               return "manifold";
+//        case EMarketplaceKind::ZORA:                   return "zora";
+//        case EMarketplaceKind::BLUR:                   return "blur";
+//        case EMarketplaceKind::SUPER_RARE:             return "super_rare";
+//        case EMarketplaceKind::OKX:                    return "okx";
+//        case EMarketplaceKind::ELEMENT:                return "element";
+//        case EMarketplaceKind::AQUA_XYZ:               return "aqua_xyz";
+//        case EMarketplaceKind::AURANFT_CO:             return "auranft_co";
+//
+//        default: return "Unknown";
+//
+//        }
+//    }
+//};
```

Comment:
What is this for?

Reply:
These are Enums function helpers for the collectibles requests enum type conversions. Just corrected and uncommented

Reply:
@BellringerQuinn, has @caballoninja answered your question?